### DISCUSSION TITLE
Is Tropical Storm And Volcanic Ash Data Available Via The API?

### DISCUSSION THREAD

**Discussion by danbo0001** [weather-gov/api#569] [Q&A] [answered]
Hi, 

I looked at the list of products available in the api using https://api.weather.gov/products/ but I couldn't find those types listed and I was wondering if maybe they are available as part of a differently named product as I think this info is available on the main weather.gov website?

The data is am looking for is:-
- tropical track forecast, probability, wind speed and rain
- volcanic ash current and forecast

If the data is available, is it US only or does it have global coverage (eg for tropical, from JTWC/JMA/ABOM/IMD - for volcanic ash, from the 7 VA advisory centres worldwide).

Many Thanks

**Comment by JJKraw** [comment, marked as answer]
This may be close to what you are looking for.

https://nws.weather.gov/products/viewItem.php?selrow=813

https://www.nhc.noaa.gov/CurrentStorms.json

Unfortunately the link to the operational notification document (via the first link above) is broken.

**Reply by danbo0001** [reply]
Hi JJ, yep it is close, but most of the links in the json doc are raw text so i would need to do some text scraping. i will have a look at a few different time periods to see how/if the structure changes. it does seem then though that neither tropical layers or ash are available through the api :(
